src/model/results-check.ts

```diff
@@ -22,9 +22,18 @@ const ResultsCheck = {
       files.map(async filepath => {
         if (!filepath.endsWith('.xml')) return;
         core.info(`Processing file ${filepath}...`);
-        const fileData = await ResultsParser.parseResults(path.join(artifactsPath, filepath));
-        core.info(fileData.summary);
-        runs.push(fileData);
+        try {
+          const content = fs.readFileSync(path.join(artifactsPath, filepath), 'utf8');
+          if (!content.includes('<test-results') && !content.includes('<test-run')) {
```

Comment:
NUnit3 result does not contain the `<test-results>` element.

Reply:
Thanks for the heads up.

I haven't tested this, I should have set the PR as Draft. I quickly wrote this based on AI's suggestion yesterday. We should give it a try on the actual project before merging as well. If you do have the solution, feel free to suggest a code change:

![CleanShot 2024-11-08 at 11 46 22@2x](https://github.com/user-attachments/assets/f69f66ba-4a55-4189-a85f-55ff5b3fc5c0)

:+1:

Reply:
Thank you for your response.

> We should give it a try on the actual project before merging as well.

I tried it below:

Tests output a JUnit XML format file.

Uses @v4
https://github.com/nowsprinting/test-helper/actions/runs/11748561326
Error.

Uses @fix/add-warning-for-invalid-nunit-xml-files
https://github.com/nowsprinting/test-helper/actions/runs/11748649133
Passed. The current code serves its purpose.
<img width="696" alt="warn" src="https://github.com/user-attachments/assets/1e0d8a87-a569-49fb-996d-a4dfc19e8b88">

Reply:
```suggestion
          if (!content.includes('<test-run')) {
```

`<test-results/>` is an element contained in the NUnit2 result.
It will not be included in the NUNit3 results.